yarpl/include/yarpl/flowable/Flowable.h

```diff
@@ -337,6 +339,12 @@ auto Flowable<T>::take(int64_t limit) {
 }
 
 template <typename T>
+auto Flowable<T>::skip(int64_t offset) {
+  return Reference<Flowable<T>>(
```

Comment:
This return could be -

  `return make_ref<SkipOperator<T>>(Reference<Flowable<T>>(this), offset);`